### PR TITLE
chore(ddx): content review dedibox

### PR DESCRIPTION
**Pull request by bene2k1**
### Your checklist for this pull request

- [ ] Name your pull request according to the [contribution guidelines](https://github.com/scaleway/docs-content/blob/main/docs/CONTRIBUTING.md).

### Description

Please describe what you added or changed.

### CODE REVIEW

File: dedibox-network/ip-failover/how-to/configure-network-virtual-machine.mdx [hunk 1]
```diff
@@ -54,8 +52,7 @@ Since the release of version 18.04 (Bionic Beaver) Ubuntu has switched to Netpla
             addresses: [fail.over.ip.address/32]
             gateway4: 62.210.0.1
             nameservers:
-              addresses: [ "51.159.47.28", "51.159.47.26" ] # Replace the IP of the DNS cache server with the one located in the same physical location as your machine for optimal performances (https://www.scaleway.com/en/docs/console/account/reference-content/scaleway-network-information/#dns-cache-servers)
-
+              addresses: [ "51.159.47.28", "51.159.47.26" ]
```
Comment (bene2k1) [inline dedibox-network/ip-failover/how-to/configure-network-virtual-machine.mdx:55]:
```suggestion
              addresses: [ "51.159.47.28", "51.159.47.26" ] # Replace the IP of the DNS cache server with the one located in the same physical location as your machine for optimal performances (https://www.scaleway.com/en/docs/console/account/reference-content/scaleway-network-information/#dns-cache-servers)
```

File: dedibox-network/ip-failover/how-to/configure-network-virtual-machine.mdx [hunk 2]
```diff
@@ -85,30 +82,25 @@ Since the release of version 18.04 (Bionic Beaver) Ubuntu has switched to Netpla
...
 4. Set the DNS server information by editing the file `/etc/resolv.conf`:
     ```
-    nano /etc/resolv.conf
+    sudo nano /etc/resolv.conf
     ```
 5. Set the DNS resolver by adding the following lines to the file. Then save it and exit the text editor:
     ```
     nameserver 51.159.47.28
     nameserver 51.159.47.26
     ```
```
Comment (bene2k1) [inline dedibox-network/ip-failover/how-to/configure-network-virtual-machine.mdx:91]:
```suggestion
    ```
    <Message type="tip">
        Replace the IP of the DNS cache server with the one located in the same physical location as your machine for optimal performances (https://www.scaleway.com/en/docs/console/account/reference-content/scaleway-network-information/#dns-cache-servers)
    </Message>
```

File: dedibox-network/ip-failover/how-to/configure-network-virtual-machine.mdx [hunk 3]
```diff
@@ -1,23 +1,21 @@
 ---
 meta:
-  title: How to configure the network of a virtual machine
-  description: This page explains how to configure the network of a virtual machine.
+  title: How to configure the network of a virtual machine on a Dedibox host
+  description: Learn how to configure network settings for virtual machines on various operating systems including Ubuntu, Debian, CentOS, and Windows Server.
```
Comment (jcirinosclwy) [inline dedibox-network/ip-failover/how-to/configure-network-virtual-machine.mdx:4]:
```suggestion
  description: Learn how to configure network settings for virtual machines on various operating systems, including Ubuntu, Debian, CentOS, and Windows Server. 
```

File: dedibox-network/ip-failover/how-to/configure-network-virtual-machine.mdx [hunk 4]
```diff
@@ -1,23 +1,21 @@
...
 meta:
-  title: How to configure the network of a virtual machine
-  description: This page explains how to configure the network of a virtual machine.
+  title: How to configure the network of a virtual machine on a Dedibox host
+  description: Learn how to configure network settings for virtual machines on various operating systems including Ubuntu, Debian, CentOS, and Windows Server. 
 content:
-  h1: How to configure the network of a virtual machine
-  paragraph: This page explains how to configure the network of a virtual machine.
+  h1: How to configure the network of a virtual machine on a Dedibox host
+  paragraph: Learn how to configure network settings for virtual machines on various operating systems including Ubuntu, Debian, CentOS, and Windows Server.
```
Comment (jcirinosclwy) [inline dedibox-network/ip-failover/how-to/configure-network-virtual-machine.mdx:7]:
```suggestion
  paragraph: Learn how to configure network settings for virtual machines on various operating systems, including Ubuntu, Debian, CentOS, and Windows Server. 
```

File: dedibox-network/ip-failover/how-to/configure-network-virtual-machine.mdx [hunk 5]
```diff
@@ -85,30 +82,28 @@ Since the release of version 18.04 (Bionic Beaver) Ubuntu has switched to Netpla
...
 5. Set the DNS resolver by adding the following lines to the file. Then save it and exit the text editor:
     ```
     nameserver 51.159.47.28
     nameserver 51.159.47.26
     ```
-
-    <Message typ="tip">
-      The DNS resolvers above are operated by Scaleway. You can also set your preferred alternative DNS resolvers in the file.
+    <Message type="tip">
+      Replace the IP of the DNS cache server with the one located in the same physical location as your machine for optimal performances (https://www.scaleway.com/en/docs/console/account/reference-content/scaleway-network-information/#dns-cache-servers)
```
Comment (jcirinosclwy) [inline dedibox-network/ip-failover/how-to/configure-network-virtual-machine.mdx:93]:
```suggestion
      Replace the IP of the DNS cache server with the one located in the same physical location as your machine for optimal performance. [Check the IPs for each datacenter](/console/account/reference-content/scaleway-network-information/#dns-cache-servers)
```

File: dedibox-network/ip-failover/how-to/configure-network-virtual-machine.mdx [hunk 6]
```diff
@@ -120,7 +115,7 @@ Since the release of version 18.04 (Bionic Beaver) Ubuntu has switched to Netpla
     NETMASK=255.255.255.255
     IPADDR=fail.over.ip.address
     GATEWAY=62.210.0.1
-    DNS1=51.159.47.28
+    DNS1=51.159.47.28 # Replace the IP of the DNS cache server with the one located in the same physical location as your machine for optimal performances (https://www.scaleway.com/en/docs/console/account/reference-content/scaleway-network-information/#dns-cache-servers)
```
Comment (jcirinosclwy) [inline dedibox-network/ip-failover/how-to/configure-network-virtual-machine.mdx:118]: Is this really supposed to be formatted like this?

File: dedibox-network/ip-failover/how-to/configure-network-virtual-machine.mdx [hunk 7]
```diff
@@ -146,4 +141,6 @@ Since the release of version 18.04 (Bionic Beaver) Ubuntu has switched to Netpla
     * DNS 1: `51.159.47.28`
     * DNS 2: `51.159.47.26`
 
-
+    <Message type="tip">
+      Replace the IP of the DNS cache server with the one located in the same physical location as your machine for optimal performances (https://www.scaleway.com/en/docs/console/account/reference-content/scaleway-network-information/#dns-cache-servers)
```
Comment (jcirinosclwy) [inline dedibox-network/ip-failover/how-to/configure-network-virtual-machine.mdx:145]:
```suggestion
      Replace the IP of the DNS cache server with the one located in the same physical location as your machine for optimal performance. [Check the IPs for each datacenter](/console/account/reference-content/scaleway-network-information/#dns-cache-servers)
```

**Review by jcirinosclwy — APPROVED** [review]

**Review by ldecarvalho-doc — APPROVED** [review]

**Pull request merged**